README.md

```diff
@@ -36,6 +43,20 @@ This should open up the website in your browser in ``http://localhost:8080``
 
 The hot script will hot reload whenever you make changes in your files so that you can see your changes right away.
 
+## Docker Start
+
+Requirements for Windows OS
+- Windows 10 Pro
+- Docker
+- Hyper-V enabled
+
+1. Windows OS: Launch docker-compose
+```powershell
+launch_docker.ps1
+```
```

Comment:
@alex-gagnon Should add instructions of OSX

Reply:
Also, add that running with docker is an alternative way to run the app locally. Explain the advantages and disadvantages.

Add instructions on how to install and run docker